content/docs/interface/example0/_index.md

```diff
@@ -1105,3 +1105,221 @@ write:
+  default_input_namespace: username
+  default_output_namespace: username
+  write_data_store: test_datastore/
+  local_repo: local_repo
+  script: |-
+    R -f simple_working_examples/output_globbing.R ${{CONFIG_DIR}}
+write:
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/1
+  description: A csv file
+  file_type: csv
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/thing/1
+  description: A csv file
+  file_type: csv
+```
+
+#### Working *config.yaml*
+
+```yaml
+run_metadata:
+  description: Register a file in the pipeline
+  local_data_registry_url: http://localhost:8000/api/
+  remote_data_registry_url: https://data.scrc.uk/api/
+  default_input_namespace: username
+  default_output_namespace: username
+  write_data_store: test_datastore/
+  local_repo: local_repo
+  script: |-
+    R -f simple_working_examples/output_globbing.R /Users/SoniaM/datastore/coderun/20210511-231444/
+  public: yes
+  latest_commit: 2a8688677321b99e3a2545ce020992d136334b71
+  remote_repo: https://github.com/fake_org/fake_repo
+read: []
+write:
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/1
+  description: A csv file
+  file_type: csv
+  use:
+    version: 0.0.1
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/thing/1
+  description: A csv file
+  file_type: csv
+  use:
+    version: 0.0.1
+```
+
+Now that our local registry is populated, we can try globbing:
+
+#### User written *config.yaml*
+
+```yaml
+run_metadata:
+  description: Register a file in the pipeline
+  local_data_registry_url: http://localhost:8000/api/
+  remote_data_registry_url: https://data.scrc.uk/api/
+  default_input_namespace: username
+  default_output_namespace: username
+  write_data_store: test_datastore/
+  local_repo: local_repo
+  script: |-
+    R -f simple_working_examples/output_globbing.R ${{CONFIG_DIR}}
+write:
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/*
+  description: A csv file
+  file_type: csv
+  use:
+    version: ${{MAJOR}}
+```
+
+#### Working *config.yaml*
+
+````yaml
+run_metadata:
+  description: Register a file in the pipeline
+  local_data_registry_url: http://localhost:8000/api/
+  remote_data_registry_url: https://data.scrc.uk/api/
+  default_input_namespace: username
+  default_output_namespace: username
+  write_data_store: test_datastore/
+  local_repo: local_repo
+  script: |-
+    R -f simple_working_examples/output_globbing.R /Users/SoniaM/datastore/coderun/20210511-231444/
+  public: yes
+  latest_commit: f95815976cd4d93c062f94a48525fcec88b6ef34
+  remote_repo: https://github.com/fake_org/fake_repo
+read: []
+write:
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/*
+  description: A csv file
+  file_type: csv
+  use:
+    version: 1.0.0
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/thing/1
+  description: A csv file
+  file_type: csv
+  use:
+    version: 1.0.1
```

Comment:
A major version bump just goes to the next whole number:
```suggestion
    version: 1.0.0
```

Reply:
oh no! I'll need to fix my code 😅